pkg/apis/jvmbuildservice/v1alpha1/systemconfig_types.go

```diff
@@ -44,3 +44,10 @@ type SystemConfigList struct {
 	metav1.ListMeta `json:"metadata,omitempty"`
 	Items           []SystemConfig `json:"items"`
 }
+
+const (
+	KonfluxGitDefinition          = "https://raw.githubusercontent.com/konflux-ci/build-definitions/refs/heads/main/task/git-clone/0.1/git-clone.yaml"
+	KonfluxPreBuildDefinitions    = "https://raw.githubusercontent.com/rnc/jvm-build-service/main/deploy/tasks/pre-build.yaml"
+	KonfluxBuildDefinitions       = "https://raw.githubusercontent.com/konflux-ci/build-definitions/refs/heads/main/task/buildah-oci-ta/0.2/buildah-oci-ta.yaml"
+	KonfluxMavenDeployDefinitions = "https://raw.githubusercontent.com/rnc/jvm-build-service/main/deploy/tasks/maven-deployment.yaml"
```

Comment:
@tecarter94 I realise this conflicts somewhat with your proxy PR. The problem is Konflux are constantly committing to the `buildah-oci-ta` task and it has diverged since I forked it. This PR removes the final modifications I made to the fork isolating them into the JBS code. 
I think what is best **after** this, is to refork / update the buildah task again (and update the above `KonfluxBuildDefinitions`) so that we're running with a recent copy.

Reply:
Cheers. It might be a while till I proceed with #2153, so I'll revisit this at that point.